Tchap/Config/BuildSettings.swift

```diff
@@ -425,6 +427,10 @@ final class BuildSettings: NSObject {
     
     static let locationSharingEnabled = false // Currently disabled in Tchap.
 
+    // MARK: - Voice Broadcast
+    static let voiceBroadcastChunkLength: Int = 120
+    static let voiceBroadcastMaxLength: UInt = 14400 // 240min.
+
```

Comment:
Please add this change in Btchap and DevTchap `BuildSettings.swift` files too.

Reply:
Done